src/components/layout/RightSectionWallet.vue

```diff
@@ -175,7 +175,7 @@ export default {
     display: block;
     border: none;
     width: 100%;
-    height: 470px;
+    height: 600px;
```

Comment:
I didn't get why we have been asked to do it. The only problem that I see is that "Terms Privacy  Branded by Venture" is hidden and can't be scrolled to at 1920x1080 screen:
![Screenshot 2021-04-09 at 02 08 10](https://user-images.githubusercontent.com/9007851/114106897-7a786300-98d8-11eb-8406-bc441f4df88f.png)

Reply:
It's because the wallet dimensions have changed, now it has 600px height.